web/scripts/retrieve_data.py

```diff
@@ -75,10 +75,9 @@
 CURRENT_YEAR = _curr_time.year if _curr_time.month > 6 else _curr_time.year - 1
 YEARS = list(range(CURRENT_YEAR - 10, CURRENT_YEAR + 1))
 
-GROUP_NAMES_TO_KEYS = {
-    "S&P 500": "sp500",
-    "Fortune Global 500": "global500"
-}
+# Used (along with a check that we never actually meet or exceed this number with legitimate CSET ids)
+# when creating fake CSET ids for groups
+GROUP_OFFSET = 1_000_000
```

Comment:
Could this get added to the `overall_data.json` file?  That would let the company/group column formatter function recognize when an entry is a group without risking breaking things in the future.

Reply:
Done, I went ahead and integrated it into table_columns.js